pkg/apb/svc_acct.go

```diff
@@ -86,11 +86,7 @@ func (s *ServiceAccountManager) createResources(rFilePath string, namespace stri
 	return nil
 }
 
-func (s *ServiceAccountManager) writeResourceFile(
-	handle string,
-	svcAcctM *map[string]interface{},
-	roleBindingM *map[string]interface{},
-) (string, error) {
+func (s *ServiceAccountManager) writeResourceFile(handle string, svcAcctM *map[string]interface{}, roleBindingM *map[string]interface{}) (string, error) {
```

Comment:
I don't think golint should fail on long signatures. I usually write the long signature but I do not rewrite the shorter one if it already exists. I don't think we should fail on this. If we want to fail, then let's fail on LONG ones. I.e. make it shorter.